Generative AI with sub-concepts (#2043)
Codecademy/docs: content/ai/concepts/generative-ai/concepts/music/music.md

---
Title: 'Music'

> Description: GenAI allows for the creation of long-playing high-fidelity music from text descriptions and additional sound input conditions.

Reviewer: ```suggestion
Description: 'GenAI allows for the creation of long-playing high-fidelity music from text descriptions and additional sound input conditions.'
```

Subjects:
  - 'Machine Learning'
  - 'Data Science'

>   - 'Large Language Model'

Reviewer: ```suggestion
```

>   - 'Transformers'

Reviewer: ```suggestion
```

> Tags:

Reviewer: Only AI is listed on the [Tags list](https://github.com/Codecademy/docs/blob/main/documentation/tags.md).
All the other terms should be added to that file in this PR.
